drivers/hmis/app/graphql/mutations/delete_auto_exit_config.rb

```diff
@@ -0,0 +1,26 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Mutations
+  class DeleteAutoExitConfig < CleanBaseMutation
+    argument :id, ID, required: true
+
+    field :auto_exit_config, Types::HmisSchema::AutoExitConfig, null: true
+    field :errors, [Types::HmisSchema::ValidationError], null: false, resolver: Resolvers::ValidationErrors
+
+    def resolve(id:)
+      record = Hmis::AutoExitConfig.find(id)
+      return { errors: [HmisErrors::Error.new(:auto_exit_config, :not_allowed)] } unless allowed?(permissions: [:can_configure_data_collection])
```

Comment:
```suggestion
      raise 'Access denied' unless allowed?(permissions: [:can_configure_data_collection])
```

We've switched to using exceptions for unexpected failures so we can track them in sentry. It looks like the `default_create_record` is still returning a validation which is maybe where you copied this from . You can update that to an exception too.